capstone/src/test/java/com/google/univiz/scorecard/CarnegieSizeDegreeTest.java

```diff
@@ -1,4 +1,4 @@
-package com.google.univiz;
+package com.google.univiz.scorecard;
```

Comment:
According to PR description, this should be com.google.univiz.api.representation.